README.md

```diff
@@ -32,8 +32,8 @@ $ cat > /etc/cdi/vendor.json <<EOF
       "name": "myDevice",
       "ociEdits": {
         "deviceNodes": [
-          {"hostPath": "/dev/card1", "containerPath": "/dev/card1"}
-          {"hostPath": "/dev/card-render1", "containerPath": "/dev/card-render1"}
+          {"path": "/dev/card1", "type": "c", "major": 25, "minor": 25, "permissions": "rw", "uid": 1000, "gid": 1000}
```

Comment:
The code seems to indicate that this is still `hostPath`. Should that be updated there too? Do we have to support both?

Reply:
It was a mistake. I had no intantion to change host* attributes. Should be fixed now.